### ISSUE TITLE
Reported wrong line number for an error

### ISSUE THREAD

**Issue by Dade916** [browndeer/coprthr#10]
The current implementation seems to report a wrong line number for any error in a kernel. It looks like a constant offset: it is always the right line number +2.

**Comment by browndeer** [comment]
Yes this has annoyed me for a while.  Your kernel is having two includes added.  Solution is known but not propagated yet.  Its actually easy, we need to use --include flag instead of modifying the kernel source.    

Sent via BlackBerry by AT&T

-----Original Message-----
